javaagent-internal-logging-application/src/main/java/io/opentelemetry/javaagent/logging/application/ApplicationLoggingCustomizer.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.logging.application;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
+import io.opentelemetry.javaagent.bootstrap.InternalLogger;
+import io.opentelemetry.javaagent.bootstrap.logging.ApplicationLoggerBridge;
+import io.opentelemetry.javaagent.tooling.LoggingCustomizer;
+
+@AutoService(LoggingCustomizer.class)
+public final class ApplicationLoggingCustomizer implements LoggingCustomizer {
+
+  @Override
+  public String name() {
+    return "application";
+  }
+
+  @Override
+  public void init() {
+    int limit =
+        ConfigPropertiesUtil.getInt("otel.javaagent.logging.application.logs-buffer-size", 2048);
```

Comment:
I thought this was bytes initially, maybe rename to something like `logs-buffer-max-records`?